ktor-io/jvm/src/io/ktor/utils/io/Delimited.kt

```diff
@@ -52,11 +53,18 @@ suspend fun ByteReadChannel.skipDelimiter(delimiter: ByteBuffer) {
     }
 }
 
-private suspend fun ByteReadChannel.skipDelimiterSuspend(delimiter: ByteBuffer) {
+private suspend fun ByteReadChannel.skipDelimiterSuspend(delimiter: ByteBuffer): Boolean {
```

Comment:
this change also requires test

Reply:
The behaviour is not changed, only a result flag is provided. Also, this is not a separate function but a companion function for tail-suspend `skipDelimiter`. We actually don't need this code anymore, since we copied it.

Reply:
Do we have a test checking that method?

Reply:
I'll remove this change but apply a test to the copied version with change

Reply:
Fixed